internal/tools/monitor/core/metric/storage/elasticsearch/query_test.go

```diff
@@ -54,7 +54,7 @@ func TestSearch(t *testing.T) {
 				expect.Sources().Return([]*model.Source{{Database: "db", Name: "name"}})
 				expect.Timestamp().Return(int64(0), int64(1))
 				expect.AppendBoolFilter(gomock.Any(), gomock.Any()).Return()
-				expect.Debug().Return(false)
+				expect.Return(false)
```

Comment:
this `Debug` is used to debug query sql, no need to delete this

Reply:
you are awesome!